### DISCUSSION TITLE
Welcome to AI-Interview Discussions!

### DISCUSSION THREAD

**Discussion by aryasoni98** [aryasoni98/AI-Interview#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hubblehox** [comment]
Hi Arya,

We're building an AI Interview module
I saw your github repo,
Can we have a discussion ?

-- 
Best,
Souvik Roy
Mob : +91-7980991392
Skype : souvikroy1
